types/vector2js/index.d.ts

```diff
@@ -0,0 +1,403 @@
+// Type definitions for vector2js 2.0
+// Project: https://github.com/RonenNess/Vector2js
+// Definitions by: Leo Nicolle <https://github.com/Leo-Nicolle>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+export = Vector2;
+
+type inputVec = Vector2 | { x: number; y: number };
+declare class Vector2 {
```

Comment:
```suggestion
declare class Vector {
```
the symbol is named `Vector`